portfolio/src/main/webapp/scripts/survey.js

```diff
@@ -0,0 +1,76 @@
+const subgenres = ['Body Horror', 'Slasher', 'Monster', 'Paranormal', 'Psychological Thriller'];
+const optionsContainer = document.getElementById('options-container');
+createSurvey();
```

Comment:
Put the executed code at the bottom.

Reply:
By executed code do you mean createSurvey()? I have moved it to the bottom of the script